packages/web/pages/player/[username].tsx

```diff
@@ -453,6 +460,21 @@ export const Grid: React.FC<Props> = ({
               Cancel
             </MetaButton>
           )}
+          {changed && canEdit && (
```

Comment:
instead of adding a new button here, could we add a pop over menu with three vertical dots ? 
this kinda clutters the UI

Reply:
i thought the same, but to quote @davort :
> It is on the longer side, but 3 words is still quite conventional, IME.
Another way to go is to just use icons, with tooltips showing up on-hover → like in Gmail or Google Drive
Right now, we only display these few options, so having a longer button label, and thus a wider button, is completely fine.
I prefer to err on the side of clarity for the user in this case, as long as we have enough space to afford it 🙂

Reply:
the suggestion is to change `Default` -> `Reset to default`

Reply:
I see alright then :)